pkg/query/operators.go

```diff
@@ -25,6 +25,8 @@ const (
 	GreaterThanOperator gtOperator = "gt"
 	// GreaterThanOrEqualOperator takes two operands and tests if the left is greater than or equal the right
 	GreaterThanOrEqualOperator geOperator = "ge"
+	// Contains takes two operands and tests if the left contains the right
+	ContainsOPerator containsOperator = "contains"
```

Comment:
should be ContainsOperator instead ContainsOPerator?

Reply:
its O not null, it looks like null but is not

Reply:
```suggestion
	ContainsOperator containsOperator = "contains"
```

Reply:
ahh sorry you mean P